### PR TITLE
Responsive Next web clone

### PR DESCRIPTION
**Pull request by NtwaliHeritier**
We cloned the next web page using: 
- grids to design the article section
- flexboxes to design the navbar
- floats to move an icon to the right in the footer
- media queries to make the webpage responsive on different devices
- fontawesome to include some icons in the web page.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,608 @@
...
+
+@media screen and (max-width: 600px) {
+  main {
+    margin: 0;
+    padding: 0;
+  }
+
+  body {
+    overflow-x: hidden;
+    width: 100%;
```
Comment (expjazz) [inline index.css]: - [ ] As the requirements in pathwright state, you should use the mobile-first approach, that means that the default is the small screen and no media queries are used for this size, next you should state the media queries for tablet @media screen and (min-width: 768px) and lastly the media queries for desktop @media screen and (min-width: 1024px). [You did desktop-first and established max-widths for your media queries]. Please correct this.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,968 @@
...
+            <img src="./images/image4.png" alt="image" />
+
+            <div class="second">
+              <p>
+                <a href="#"
+                  >This Mars helicopter will be the first ever to fly on another
+                  planet</a
+                >
+              </p>
+              <p class="second-p"><a href="#">Rachel Kaser</a> - 13hours ago</p>
```
Comment (expjazz) [inline index.html]: #### Mobile Screen Size

- [ ] The author's name here is with the wrong styling.

- [ ] The image can be smaller

| Yours | Original |
|:---:|:---:|
|image| image  |

File: index.html [hunk 3]
```diff
@@ -0,0 +1,968 @@
...
+            <h1>Latest funding rounds on Index.co</h1>
+            <i class="fas fa-angle-right"></i>
+          </div>
+          <div>
+            <a class="red" href="#">Add info |</a
+            ><a class="red" href="#"> Get updates by email |</a
+            ><a class="red" href="#"> See more</a>
+          </div>
+        </div>
+
```
Comment (expjazz) [inline index.html]: #### Mobile Screen Size

- [ ] The layout of these links is different from the original

| Yours | Original |
|:---:|:---:|
|image | image |

File: index.html [hunk 4]
```diff
@@ -0,0 +1,968 @@
...
+        <div class="grid section3">
+          <div class="majordiv">
+            <h1>Amwell</h1>
+            <div class="div1">
+              <p>$100M</p>
+              <p class="grey">CORPORATE ROUND</p>
+            </div>
+            <div class="div2">
```
Comment (expjazz) [inline index.html]: #### Mobile Screen Size

- [ ] The text on the right side of the image should not be centralized. Should be closer to the image.

| Yours | Original |
|:---:|:---:|
|image |image |

File: index.html [hunk 5]
```diff
@@ -0,0 +1,968 @@
...
+              <p class="grey lastp">on Pulse 2.0</p>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- section 4 -->
+      <section class="grid2 alignment section4">
+        <div>
+          <div class="title">
+            <h1>Latest news</h1>
```
Comment (expjazz) [inline index.html]: - [ ] I know you did not worried about the content, but it is confusing to see the same title from a previous section again

File: index.html [hunk 6]
```diff
@@ -0,0 +1,968 @@
...
+            <h1>Latest Deals</h1>
+            <i class="fas fa-angle-right"></i>
+          </div>
+          <div>
+            <a class="red" href="#">See all deals</a>
+          </div>
+        </div>
+        <div class="grid grid-divs">
+          <div>
+            <img src="./images/suit.jpg" alt="image" />
```
Comment (expjazz) [inline index.html]: #### Mobile Screen Size

- [ ] Here the layout is different from the original. Kindly address that.

| Yours | Original |
|:---:|:---:|
|image | image |

File: index.html [hunk 7]
```diff
@@ -0,0 +1,968 @@
...
+            </p>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer>
+      <div class="ftr1 sectionflex">
+        <div class="links">
+          <a href="#"><i class="fab fa-facebook fa-2x"></i></a>
+          <a href="#"><i class="fab fa-twitter fa-2x"></i></a>
```
Comment (expjazz) [inline index.html]: #### Mobile Screen Size

- [ ] The bottom of the footer needs to be different

| Yours | Original |
|:---:|:---:|
|image|image |

**Review by expjazz — CHANGES_REQUESTED** [review]
## Status: Changes Required :recycle: 

Hello :handshake: 

You did a very good job on this project. The design looks great, and I can see that you have great attention to detail :clap:  Your code is also clan and well structured :100: 

However, there are a few mistakes that we need to fix in order to move on:

- [ ] It seems that you did not change the layout of the page on the medium screen size. Kindly address that. Your clone should reflect the same items per columns and number of rows of the original.

:arrow_down: 

| Yours | Original |
|:---:|:---:|
|image | image |

- [ ] Check the inline comments for more.

#### You should be proud. The page already looks really great!

### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:_

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi Team! 👋 
### Great job implementing the requested changes!! 🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****

 Good Luck and Happy Coding! 🍀 👩🏾‍💻

**Pull request merged**